strawberry/ext/mypy_plugin.py

```diff
@@ -1007,7 +1011,10 @@ def _is_strawberry_pydantic_decorator(self, fullname: str) -> bool:
 
 
 def plugin(version: str):
-    # Save the version to be used by the plugin.
-    MypyVersion.VERSION = Decimal(version)
+    match = VERSION_RE.match(version)
+    if match:
+        MypyVersion.VERSION = Decimal(".".join(match.groups()))
+    else:
+        MypyVersion.VERSION = FALLBACK_VERSION
```

Comment:
is this safe to do? should we at least add a warning?

Reply:
Good suggestion, a warning would be nice, I'll add it!

Safety-wise, at worst-case certain features won't be correctly enabled for the required version, but `mypy` will still attempt checking the code, instead of directly failing 😊

Reply:
gotcha!